app.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express")
+const app = express()
+const { getTopics } = require("./controlllers/ncNewsController")
+const {handle404Statuses, handle500Statuses} = require("./controlllers/errorHandlingController")
+app.use(express.json())
+
+app.get("/api/topics", getTopics)
+
+
+app.all('*', (req, res, next)=>{
+    next({status:404})
```

Comment:
This should be res.status(404).send({message: "404 path not found"}) - or something similar.